orttraining/orttraining/test/python/orttraining_distributed_tests.py

```diff
@@ -14,11 +14,26 @@
     level=logging.DEBUG)
 log = logging.getLogger("DistributedTests")
 
+
```

Comment:
Refactor into https://github.com/microsoft/onnxruntime/tree/master/orttraining/orttraining/test/python/checkpoint
to keep all checkpoint related tests together